lesson-demo/src/main/resources/hw19_script.sql

```diff
@@ -0,0 +1,81 @@
+create schema hw19;
+set schema 'hw19';
+
+create table "user"
+(
+    id       bigserial,
+    email    varchar(100) not null,
+    username varchar(100) not null,
+    constraint user_pk primary key (id),
+    constraint user_email_uq unique (email),
+    constraint user_username_uq unique (username)
+);
+
+create type "advisor_role" as enum ('associate', 'partner', 'senior');
+
+create table "advisor"
+(
+    user_id bigint,
+    role    advisor_role not null,
+    constraint advisor_pk primary key (user_id),
+    constraint advisor_user_fk foreign key (user_id) references "user"
+);
+
+create table "applicant"
+(
+    user_id    bigint,
+    first_name varchar(100) not null,
+    last_name  varchar(100) not null,
+    ssn        bigint       not null,
+    constraint applicant_pk primary key (user_id),
+    constraint applicant_user_fk foreign key (user_id) references "user",
+    constraint applicant_ssn_uq unique (ssn)
+);
+
+create type "application_status" as enum ('new', 'assigned', 'on_hold', 'approved', 'canceled', 'declined');
+
+create table "application"
+(
+    id           bigserial,
+    amount       int                not null,
+    status       application_status not null,
+    created_at   timestamp          not null default now(),
+    assigned_at  timestamp,
+    applicant_id bigint             not null,
+    advisor_id   bigint,
+    constraint application_pk primary key (id),
+    constraint application_applicant_fk foreign key (applicant_id) references applicant,
+    constraint application_advisor_fk foreign key (advisor_id) references advisor
```

Comment:
It is better to follow a common convention for project. I talk about how you reference to another table with quotes above and without quotes below.